src/booking/containers/booking-check/booking-check.component.ts

```diff
@@ -61,6 +61,16 @@ export class BookingCheckComponent implements OnInit {
         ));
       }
     }
```

Comment:
Unit tests?

Reply:
Changes pushed. Can you please verify again?